tests/common/startup/configuration_builder.rs

```diff
@@ -0,0 +1,67 @@
+#![cfg(feature = "integration-test")]
+
+use common::jcli_wrapper;
+
+use common::configuration::{
+    genesis_model::{Fund, GenesisYaml},
+    jormungandr_config::JormungandrConfig,
+    node_config_model::{NodeConfig, Peer},
+    secret_model::SecretModel,
+};
+
+pub struct ConfigurationBuilder {
+    funds: Vec<Fund>,
+    trusted_peers: Option<Vec<Peer>>,
+    block0_hash: Option<String>,
+}
+
+impl ConfigurationBuilder {
+    pub fn new() -> Self {
+        ConfigurationBuilder {
+            funds: vec![],
+            trusted_peers: None,
+            block0_hash: None,
+        }
+    }
+
+    pub fn with_funds<'a>(&'a mut self, funds: Vec<Fund>) -> &'a mut Self {
```

Comment:
Good. Conventionally, there's no need for `with_` prefix in builder-style methods, just the names like members.

Reply:
You can omit the lifetime parameter in this case, it is inferred.